scripts/aws/setupdb.sh

```diff
@@ -44,6 +45,8 @@ while getopts ":hbsdpmqf:" opt; do
             load_water_quality=true ;;
         f)
             file_to_load=$OPTARG ;;
+        x)
```

Comment:
I think that `x:` needs to be added to `getopts` above or else I get the following error message:

```
invalid option: -x
```